### DISCUSSION TITLE
myblog/icon-computer/

### DISCUSSION THREAD

**Discussion by giscus** [jasoneckert/jasoneckert.github.io#1] [Announcements]
# myblog/icon-computer/

My personal website and blog

https://jasoneckert.github.io/myblog/icon-computer/

**Comment by jasoneckert** [comment]
I've received 94 emails about the ICON (and LEXICON) since this blog post was published. Now that I've added a comment section, people can add their comments here instead (however, emails are always welcome too!)

**Comment by 10thfloor** [comment]
This computer made a HUGE impression on me. In elementary school we had a bank of 5 of these, I used to skip lunch recess and go to the library and create drawings and animations using the painter software, and play Oregon trail ...

**Comment by mikabre** [comment]
This computer system is why I work in tech today. Of course, I only had access to them in elementary school, so I mostly goofed off and played games on them. It's absolutely criminal that they were ordered destroyed, I wish I could get my hands on a ICON/LEXICON pair to tinker with.

**Comment by Shadyman** [comment]
Interesting. We had the next generation ones whose Lexicon was in a slimmed down chassis with half height 5.25 floppy and HDD, I believe. 

The next-gen Icons were all built into the screen with a keyboard and trackball on a coiled cable to the terminal.

I think they shared the same ARCNet setup.

I have a pamphlet somewhere for parents that extolled all the various software that was available for docent grade levels.

**Comment by mijk84** [comment]
I'm looking for disk images in order to emulate an ICON/LexICON

**Comment by LaszloQPanaflex** [comment]
I have some great memories of using these for several years through both elementary and high school. I think The Bartlett Saga series of games was one of my favorite, along with Offshore Fishing of course. I would love to be able to play those again.
